glidertest/tools.py

```diff
@@ -324,6 +370,10 @@ def sunset_sunrise(time, lat, lon):
     sunset: numpy.ndarray
         An array of the sunset times.
 
+    Original author
+    ----------------
+    Function from GliderTools

```

Comment:
Link to this I think it's 

https://github.com/GliderToolsCommunity/GliderTools/blob/9e4c287ba080385e3a7774bf4f375365f5b05d62/glidertools/optics.py#L409

?

Reply:
I have the link already in the top description. Shall I repeat it?

Reply:
ah my bad! I think moving it here would make sense for consistency though? So that all authorship/references are in the same place across functions